src/test/java/com/hotels/bdp/circustrain/bigquery/client/BigQueryMetastoreClientTest.java

```diff
@@ -255,4 +269,21 @@ public void listPartitionsWithoutTableCachedAndPartitionsSizeNegativeReturnsAllP
     assertEquals(10, results.size());
     verify(cache).get(databaseName, tableName);
   }
+
+  private void setUpSchema() throws IOException {
+    byte[] content = getContentFromFileName("usa_names.avro");
+
+    when(extractionService.getStorage()).thenReturn(storage);
+    when(storage.list(anyString(), any(BlobListOption.class), any(BlobListOption.class))).thenReturn(blobs);
+    when(blobs.iterateAll()).thenReturn(Arrays.asList(blob));
+    when(blob.getContent()).thenReturn(content);
+    schema = new String(getContentFromFileName("usa_names_schema.avsc"));
+  }
+
+  private byte[] getContentFromFileName(String name) throws IOException {
+    File file = new File("src/test/resources/" + name);
```

Comment:
Rather than hard code paths to file locations use the lastcommons-test library (https://github.com/lastfm/lastcommons-test) which we use in quite a few of our projects. You can create a "class level" test folder and refer to files under it using a JUnit rule.